### DISCUSSION TITLE
Azure Devops WIKI and MKDOCS - How to create a excellent navigation?

### DISCUSSION THREAD

**Discussion by Dirk-michiels** [squidfunk/mkdocs-material#4232] [General]
We are using [Microsoft Azure Devops Wiki](https://docs.microsoft.com/en-us/azure/devops/project/wiki/add-edit-wiki?view=azure-devops&tabs=browser) for our product documentation.  This allows us to created Mark Down Files very easily. The MD-files also have a hierarchical relationship. We are though struggling in setting up the left side menu structure / navigation in the site created by MKDOCS.

The structure we have in mind is something like [[this](https://microsoft.github.io/code-with-engineering-playbook/documentation/recipes/static-website-with-mkdocs/) ](https://microsoft.github.io/code-with-engineering-playbook/documentation/recipes/static-website-with-mkdocs/) We can create this in Azure Develop WIKI and you can navigate it there, but we can not get MKDOCS to create the same navigation structure. The ordering (.Ordering is there) is not there any more and apparently H1 in the MD files are used as document titles in the menu. We are clearly missing out on something.

We think we can add the structure in the MKDOCS YML file but this this a painstaking process with many file and hope this is not necessary.
 
Is there a good resource on how to do this?

All pointers - very much appreciated.

Best Regards

Dirk

Here is our setup:-

- **the Mkdoc.yml file**

![mkdocsyml](https://user-images.githubusercontent.com/24853585/183870081-625e6a5b-bbff-47b9-8666-86c8979aa8f7.png)

**- the structure in Azure Devops**

![Azure_devops_Wiki](https://user-images.githubusercontent.com/24853585/183870117-7786bd41-c36b-4428-b972-732a3f1df721.png)

**- Repo Files**

![Repo_Files](https://user-images.githubusercontent.com/24853585/183870156-c411e321-097e-4c48-ac4a-5106be17d11f.png)

- The MKDocs output

![MKDOCS_Generated](https://user-images.githubusercontent.com/24853585/183870333-cbed2594-e1fd-438d-ba02-67b33636ee82.png)

**Comment by squidfunk** [comment]
Good question, I'm not proficient in Azure. Maybe the `mkdocs-awesome-pages-plugin` can help?

**Comment by jromal** [comment]
Why you just do not create a repository for the documentation? And do not use DevOps Wiki?
